blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -430,6 +450,26 @@ private MetricData getStringMetricDataForRange(Locator locator, Range range, Gra
         return new MetricData(points, getUnitString(locator), MetricData.Type.STRING);
     }
 
+    private MetricData getEnumMetricDataForRange(final Locator locator, Range range, Granularity gran, RollupType rollupType, DataType dataType) {
```

Comment:
I am not sure whether we need the hashes 2 string in getDataToRoll. I am looking into that currently. I think the method is used only for rollups.